### PR TITLE
Convert examples/components/pageviewer.js to await/async (issue 14127)

### LINKED ISSUE #14127: Convert examples/components/pageviewer.js to await/async
**Please do not work on this issue unless you have applied to the outreachy project.**

The [pageviewer.js](https://github.com/mozilla/pdf.js/blob/master/examples/components/pageviewer.js) example currently uses the older style of `promise.then` and callbacks. Convert it to use an `async` function and `await` style.

### PR DESCRIPTION
**Pull request by SaiKiranMukka**
resolves #14127

### CODE REVIEW

File: examples/components/pageviewer.js [hunk 1]
```diff
@@ -47,28 +47,29 @@ const loadingTask = pdfjsLib.getDocument({
   cMapPacked: CMAP_PACKED,
   enableXfa: ENABLE_XFA,
 });
-loadingTask.promise.then(function (pdfDocument) {
+
```
Comment (Snuffleupagus) [inline examples/components/pageviewer.js:50]: Given that the existing code doesn't have a blank-line here, let's not add one either.

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]
The beginning of the commit message feels a little redundant, with both of the words "refactor" and "convert" used, hence please change it to the following instead (with correct initial capitalization):
```
Convert examples/components/pageviewer.js to await/async (issue 14127)
```

When updating the patch, please keep https://github.com/mozilla/pdf.js/wiki/Squashing-Commits in mind.

File: examples/components/pageviewer.js (hunk 1 above)
Comment (SaiKiranMukka) [reply examples/components/pageviewer.js]: Updated

**Review by Snuffleupagus — APPROVED** [review]
r=me, thanks for the patch!

**Pull request merged**